20180321/README.md

```diff
@@ -1 +1,188 @@
-# From 0 to Match - Set Up Everything with Apple Xcode
+# First mobile app with Matchmore
+
+In this post, we will go through setting up an iOS application with AlpsSDK, from registration on the portal to obtaining a match.
+
+## Before you start
+We will use cocoapods, to install it execute in terminal:
```

Comment:
`CocoaPods`

Reply:
`In order to install this package manager you'll need to execute this command in terminal:`